### PR TITLE
Renamed PushType.FullCRUD to FullPush

### LINKED ISSUE #206: Rename `PushType.FullCRUD`  to `PushType.FullPush`
After discussion wiht @al-fisher and @IsakNaslundBh 

to improve user experience.

It will ripple through to:

- Revit: https://github.com/BHoM/Revit_Toolkit/pull/608
- ETABS: https://github.com/BHoM/ETABS_Toolkit/pull/285

### PR DESCRIPTION
**Pull request by alelom**
### Issues addressed by this PR

Closes #206 

## NOTE: Dependant branches:
Adding do-not-merge because it will ripple through to:
- Revit: https://github.com/BHoM/Revit_Toolkit/blob/713433f96d56d397ebd048114bd0238534515311/Adapter_Revit_UI/AdapterActions/Push.cs
- ETABS: https://github.com/BHoM/ETABS_Toolkit/blob/42ed85347966bcfd907f62b01d250348870037c7/Etabs_Adapter/CRUD/Create/_Create.cs)

### CODE REVIEW

**Comment by alelom** [comment]
This might ripple to Revit, adding do-not-merge

**Comment by alelom** [comment]
@pawelbaran do you think we can push this through in your open PR? It would be great to get this done ASAP. I can help if needed.

**Comment by pawelbaran** [comment]
@alelom think it would be quite an effort to add dependent branch, pair it so that CI does not complain etc. (particularly as @FraserGreenroyd is on annual leave). But my ongoing PR is ready and I am happy to merge it as soon as you and @kThorsager approve - this could be resolved straight after (even today). Would that be fine?

**Comment by alelom** [comment]
> @alelom think it would be quite an effort to add dependent branch, pair it so that CI does not complain etc. (particularly as @FraserGreenroyd is on annual leave). But my ongoing PR is ready and I am happy to merge it as soon as you and @kThorsager approve - this could be resolved straight after (even today). Would that be fine?

I think so, this was my proposal as well - as long as CI doesn't complain and let us do our job! 😄

**Comment by IsakNaslundBh** [comment]
Could we make sure we merge the other open PR before this, as that one has more changes in it (and this is a simple name change)

**Comment by alelom** [comment]
> Could we make sure we merge the other open PR before this, as that one has more changes in it (and this is a simple name change)

Yes @IsakNaslundBh plan has changed and that's what we are currently doing. @pawelbaran to confim

**Comment by pawelbaran** [comment]
Fine for me - I will open a PR that aligns Revit_Toolkit to this one, but we can keep it hanging so no prob on my end.

**Comment by FraserGreenroyd** [comment]
> I think so, this was my proposal as well - as long as CI doesn't complain and let us do our job! 😄

You do your job properly and my little CI friends won't disturb you :wink: 😄

**Comment by alelom** [comment]
> > I think so, this was my proposal as well - as long as CI doesn't complain and let us do our job! 😄
> 
> You do your job properly and my little CI friends won't disturb you 😉 😄

Yes, [sure](https://github.com/BHoM/Test_Toolkit/issues/188). 
(Just teasing you 😄 )

**Comment by FraserGreenroyd** [comment]
> > > I think so, this was my proposal as well - as long as CI doesn't complain and let us do our job! 😄
> > 
> > 
> > You do your job properly and my little CI friends won't disturb you 😉 😄
> 
> Yes, [sure](https://github.com/BHoM/Test_Toolkit/issues/188).
> (Just teasing you 😄 )

Touché :smile:

File: BHoM_Adapter/AdapterActions/_PushMethods/CRUDDispatchers/UpdateOnly.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public abstract partial class BHoMAdapter
                 var dependencyObjects = Engine.Adapter.Query.GetDependencyObjects<T>(newObjects, dependencyTypes, tag); //first-level dependencies
 
                 foreach (var depObj in dependencyObjects)
-                    if (!FullCRUD(depObj.Value as dynamic, PushType.FullCRUD, tag, actionConfig))
+                    if (!FullCRUD(depObj.Value as dynamic, PushType.FullPush, tag, actionConfig))
```
Comment (pawelbaran) [inline BHoM_Adapter/AdapterActions/_PushMethods/CRUDDispatchers/UpdateOnly.cs]: Shouldn't we change `FullCRUD` method name to `FullPush` in order to keep consistent?

Comment (alelom) [reply BHoM_Adapter/AdapterActions/_PushMethods/CRUDDispatchers/UpdateOnly.cs]: Method name stays as only enums are exposed to the users which shouldn't necessarily need to know of the CRUD methodology

**Comment by alelom** [comment]
Now conflicting branch, reopening with new one

**Comment by alelom** [comment]
New one: https://github.com/BHoM/BHoM_Adapter/pull/219

**Pull request closed without merging**